envoy/api/v2/core/protocol.proto

```diff
@@ -33,6 +33,19 @@ message Http1ProtocolOptions {
   // envoy as their HTTP proxy. In Unix, for example, this is typically done by setting the
   // *http_proxy* environment variable.
   google.protobuf.BoolValue allow_absolute_url = 1;
+
+  // [#not-implemented-hide:]
+  // Handle incoming HTTP/1.0 and HTTP 0.9 requests.
+  // This is off by default, and not fully standards compliant. There is support for pre-HTTP/1.1
+  // style connect logic, dechunking, and handling lack of client host iff
+  // default_host_for_http_10 is configured.
+  bool accept_http_10 = 2;
+
+  // [#not-implemented-hide:]
+  // A default host for HTTP/1.0 requests. This is highly suggested if accept_http_10 is true on as
```

Comment:
typo "is true on"

Also, while you are in her, can you make option references italics? E.g., `*accept_http_10*` (or just ref link them)